doc/sphinx/language/extensions/implicit-arguments.rst

```diff
@@ -3,6 +3,23 @@
 Implicit arguments
 ------------------
 
+This is a term ref: :term:`function`
+
+.. glossary::
+
+   function
+
+     Any Coq object whose type is a product.
+
+
+This is the :gdef:`bar name` definition.
+
+This is the :gdef:`fizname` definition.
+
+This is a :term:`bar name` reference.  And another: :term:`bar name`
+
+This is a :term:`fizname` reference.
```

Comment:
I'm confused by this addition.  Why is it in the implicit arguments section?

Reply:
That's a test commit coming from the glossary PR. It won't be there anymore in my next push.